reporter/src/funTest/kotlin/reporters/EvaluatedModelReporterFunTest.kt

```diff
@@ -23,39 +23,68 @@ import io.kotest.core.TestConfiguration
 import io.kotest.core.spec.style.WordSpec
 import io.kotest.matchers.shouldBe
 
-import java.io.File
-
 import org.ossreviewtoolkit.model.FileFormat
+import org.ossreviewtoolkit.model.FileFormat.JSON
+import org.ossreviewtoolkit.model.FileFormat.YAML
 import org.ossreviewtoolkit.model.OrtResult
 import org.ossreviewtoolkit.model.utils.DefaultResolutionProvider
 import org.ossreviewtoolkit.reporter.ReporterInput
+import org.ossreviewtoolkit.reporter.reporters.EvaluatedModelReporter.Companion.OPTION_OUTPUT_FILE_FORMATS
 import org.ossreviewtoolkit.utils.normalizeLineBreaks
 import org.ossreviewtoolkit.utils.test.createTestTempDir
 import org.ossreviewtoolkit.utils.test.readOrtResult
 
+import java.io.File
+
+private const val TEST_FILES_PREFIX = "src/funTest/assets/evaluatedModelReporter/"
+private const val GENERAL = "general/"
+private const val VULNERABILITIES = "vulnerabilities/"
+private const val EXPECTED_OUTPUT = "expected-output"
+private const val HTML_REPORTER_INPUT = "static-html-reporter-test-input"
+
 class EvaluatedModelReporterFunTest : WordSpec({
     "EvaluatedModelReporter" should {
-        "create the expected JSON output" {
-            val expectedResult = File(
-                "src/funTest/assets/evaluated-model-reporter-test-expected-output.json"
-            ).readText()
-            val ortResult = readOrtResult("src/funTest/assets/static-html-reporter-test-input.yml")
 
-            generateReport(ortResult).normalizeLineBreaks() shouldBe expectedResult
+        "create the general expected JSON output" {
```

Comment:
Why did you decide to create a separate test asset?
e.g. instead of adding vulnerabilities to the existing one.

Reply:
It's really hard to understand what is wrong when something fails, as well as create new test when you are working with assertion about 1000 rows. I  thikn it would be better to have few separated tests for different parts and cases instead of one test for everything. Am i wrong with my thoughts?

Reply:
The approach of this test already before was testing many different things. E.g. the test asset IMO intends to cover as many use cases as possible. I prefer to not refactor this approach now, but just add the vulnerabilities to it, so that it's cover by the static html and evaluated model test. 

Moving more towards testing less things at a time might make sense in general. I believe applying that approach is a bigger topic and needs way more refactoring than done HERE. Separating out only the vulnerabilities while testing all other things together seems inconsistent to me and introduces redundancy. Also the need for testing both JSON and YAML does not exist anymore for both cases.